blocks/search-results-list-block/features/search-results-list/_children/search-result.jsx

```diff
@@ -1,79 +1,126 @@
 import React from "react";
 import getProperties from "fusion:properties";
 
-import ArticleDate from "@wpmedia/date-block";
-import { Image } from "@wpmedia/engine-theme-sdk";
-import { extractImageFromStory } from "@wpmedia/resizer-image-block";
-import { Byline, Heading, SecondaryFont } from "@wpmedia/shared-styles";
+import {
+	Attribution,
+	Conditional,
+	Date as DateComponent,
+	formatAuthors,
+	formatURL,
+	getImageFromANS,
+	Heading,
+	HeadingSection,
+	Image,
+	Join,
+	Link,
+	MediaItem,
+	Paragraph,
+	Separator,
+	usePhrases,
+	Overline,
+} from "@wpmedia/arc-themes-components";
 
-const SearchResult = ({
-	element,
-	arcSite,
-	targetFallbackImage,
-	resizedImageOptions,
-	promoElements = {},
-	imageProps,
-}) => {
-	const {
-		description: { basic: descriptionText } = {},
-		headlines: { basic: headlineText } = {},
-		display_date: displayDate,
-		websites = {},
-	} = element;
-	if (!websites[arcSite]) {
+import { localizeDateTime } from "@wpmedia/engine-theme-sdk";
+
+const SearchResult = ({ arcSite, className, content, promoElements }) => {
+	const phrases = usePhrases();
+
+	if (!content?.websites?.[arcSite]) {
 		return null;
 	}
 
-	const url = websites[arcSite].website_url;
-	const { showHeadline, showImage, showDescription, showByline, showDate } = promoElements;
+	const {
+		overlineURL,
+		showByline,
+		showDate,
+		showDescription,
+		showHeadline,
+		showImage,
+		showItemOverline,
+	} = promoElements;
+	const {
+		dateLocalization: { dateTimeFormat, language, timeZone } = {
+			dateTimeFormat: "LLLL d, yyyy 'at' K:m bbbb z",
```

Comment:
While this is being edited see this PR - https://github.com/WPMedia/arc-themes-blocks/pull/1569

This format is actually incorrect